### ISSUE TITLE
How to add new pane for technical indicators like stochastic and macd?

### ISSUE THREAD

**Issue by anurag-py** [louisnw01/lightweight-charts-python#3]
Hi, 

great work on the package. But how do i add technical indicators on  a separate pane.

**Comment by louisnw01** [comment]
Hey!

Unfortunately there is no way to directly add a new pane to the chart at the moment. The current workaround  is to create a separate `Chart` object and use its `create_line` method in your use case. For example:

```python
from lightweight_charts import Chart
import pandas as pd

if __name__ == '__main__':

    chart = Chart()
    chart2 = Chart(volume_enabled=False, height=200, x=560, y=700)

    df = pd.read_csv('ohlcv.csv')
    chart.set(df)

    df['value'] = df['close']
    line = chart2.create_line()
    line.set(df)

    chart2.show()
    chart.show(block=True)

```

I am currently working on a new feature which will add a `create_subchart` method to `Chart`, allowing for multipane charts within the same window. I'll leave this issue open until this has been implemented!

**Comment by louisnw01** [comment]
This feature is now implemented in the latest version 1.0.5. A good place to start would be the [create_subchart](https://lightweight-charts-python.readthedocs.io/en/latest/docs.html#create-subchart) method, and in your specific case of technical indicators on a seperate pane, the [synced line chart example](https://lightweight-charts-python.readthedocs.io/en/latest/docs.html#synced-line-chart-example).

Thanks for the suggestion!
